docs/src/index.scss

```diff
@@ -35,15 +35,31 @@
         right: 0;
         padding: 25px;
     }
+
     .docs-conf-remote-stream {
-        &.rcw-stream {
-            border: 5px solid rgba(0,0,0,0.1);
+        position: relative;
+
+        &__stream {
+            &.rcw-stream {
+                border: 5px solid rgba(0,0,0,0.1);
+            }
+            video {
+                width: 100%;
+                height: auto;
+            }
         }
-        video {
-            width: 100%;
-            height: auto;
+
+        &__name {
+            position: absolute;
+            bottom: 15px;
+            left: 15px;
+
+            &--is-speaking {
```

Comment:
isSpeaking?

Reply:
https://en.bem.info/methodology/naming-convention/
Names of BEM entities are written using numbers and lowercase Latin characters. Individual words within names are separated by a hyphen (-).